src/usr/lib/rsetup/cli/wi-fi.sh

```diff
@@ -3,14 +3,38 @@
 ALLOWED_RCONFIG_FUNC+=("connect_wi-fi")
 
 connect_wi-fi() {
+local i result command
     case $# in
-        1)
+        1|2)
             nmcli radio wifi on
-            nmcli dev wifi connect "$1"
-            ;;
-        2)
-            nmcli radio wifi on
-            nmcli dev wifi connect "$1" password "$2"
+
+            case $# in
+                1)
+                    command="nmcli dev wifi connect $1"
+                    ;;
+                2)
+                    command="nmcli dev wifi connect $1 password $2"
+                    ;;
+            esac
+
+            for ((i = 0; i < 10; i++))
+            do
+                eval "$command"
+
+                result=$?
+
+                if ((result == 0))
+                then
+                    echo "WiFi connection successful"
+                    return
+                else
+                    echo "Failed, trying reconnection"
```

Comment:
```suggestion
                    echo "Failed to connect to '$1'. Retry $((i + 1)) of $total_retry." >&2
```
加一个total_retry的定义，避免魔术数字